Explorer/Assets/DCL/SDKComponents/SceneUI/Systems/UICanvasInformation/UICanvasInformationSystem.cs

```diff
@@ -53,12 +57,18 @@ private void PropagateToScene()
         private void UpdateUICanvasInformationComponent()
         {
             if (lastViewportResolutionWidth == Screen.width && lastScreenRealResolutionWidth == Screen.mainWindowDisplayInfo.width)
-                return;
+            {
+                //We add this logic because in the first frames the message might get lost and we wont send correctly the size of the screen
+                //to the scene, causing breaking UIs
+                if (currentAttempts > TOTAL_ATTEMPTS) { return; }
```

Comment:
It's very worrying, we must investigate it further.
Can you create a ticket?

Reply:
Yessir! 🫡 I will try to investigate a bit more as I keep debugging issues with this scene...

Reply:
Soooo, I found the issue 🤦 
We were calling the method to update the canvas thing from the Initialize method, even before we started the proper ECS loop, so of course, this was never properly propagated to the scene 🤦
Removing that, we are good.